tests/beaker_tests/bgp/bgplib.rb

```diff
@@ -360,8 +283,7 @@ def self.create_bgp_manifest_title_pattern1
         ensure                                 => present,
         asn                                    => #{BgpLib::ASN},
       }
-    }
-    EOF"
+    }\nEOF"
```

Comment:
The temp manifests were ending up with "   EOF" as the last line, which puppet was ignoring (or warning about, can't remember).  Removing the leading spaces before "EOF" made it work as expected.